docs/sources/auth/azuread.md

```diff
@@ -20,31 +20,32 @@ The Azure AD authentication provides the possibility to use an Azure Active Dire
 
 To enable the Azure AD OAuth2 you must register your application with Azure AD.
 
-1. Log in to [Azure Portal](https://portal.azure.com) and click **Azure Active Directory** in the side menu.
+1. Log in to [Azure Portal](https://portal.azure.com) and click **Azure Active Directory** in the side menu. If you have access to more than one tenant, select your account in the upper right. Set your session to the Azure AD tenant you wish to use.
 
-1. Click **App Registrations** and add a new application registration with a fitting name.
-   
-1. Click on **Authentication** then **Add a platform** and pick **Web App**.
+1. Click **Manage**, **App Registrations** and then **New Registration**. Provide a fitting name.
 
-1. Add the redirect URL `https://<grafana domain>/login/azuread`.
+1. Under **Redirect URI**, select **Web** as the app type. 
 
-1. Click **Overview** and then **Endpoints**.
-   - Note down the **OAuth 2.0 authorization endpoint (v2)**, this will be the auth URL.
-   - Note down the **OAuth 2.0 token endpoint (v2)**, this will be the token URL.
+1. Add the redirect URL `https://<grafana domain>/login/azuread`, then click **Register**.
 
-1. Close the Endpoints page to come back to the application overview.
+1. The app's **Overview** page is displayed.
 
-1. Note down the "Application ID", this will be the OAuth client id.
+   - Note the **Application ID**, this is the OAuth client id.
+
+1. Click **Manage**, **App Registrations** and then **Endpoints** from the top menu.
```

Comment:
```suggestion
1. Click **Manage**, **App Registrations**, and then **Endpoints** from the top menu.
```

Reply:
Per the [Documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md), always use the serial comma.

Reply:
> Per the [Documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md), always use the serial comma.

Love it! I'm actually surprised I didn't automatically include that, as I am very pro-Oxford.